### PR TITLE
Payment History - Remove accordion on Debt Details page #86969

### PR DESCRIPTION
**Pull request by amponce**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)

## Summary

- Accordion removal from “How to Pay” & text updates on the Debt Details page
Add address to Pay by Mail section.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#86969

## Testing done

- manual/visual

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Desktop |  localhost_3001_manage-va-debt_summary_debt-balances (1) |    localhost_3001_manage-va-debt_summary_debt-balances_details_79612301871 (1) |

## What areas of the site does it impact?

*Combined Debt Portal Summary*

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.

### CODE REVIEW

File: src/applications/combined-debt-portal/debt-letters/components/HowDoIPay.jsx [hunk 1]
```diff
@@ -1,114 +1,94 @@
...
+          <li>File Number</li>
+          <li>Payee Number</li>
+          <li>Person Entitled</li>
+          <li>Deduction Code</li>
+        </ul>
       )}
-    </va-accordion>
-  </article>
+
+      <a className="vads-c-action-link--blue" href="https://www.pay.va.gov/">
```
Comment (josephrlee) [inline src/applications/combined-debt-portal/debt-letters/components/HowDoIPay.jsx:59]: Use the updated `<va-action-link>` component here: https://design.va.gov/storybook/?path=/docs/components-va-link-action--docs

**Review by josephrlee — CHANGES_REQUESTED** [review]
Just one minor edit! Other than that, looks good!

**Review by josephrlee — APPROVED** [review]

**Review by IGallupSoCo — APPROVED** [review]
LGTM

**Comment by amponce** [comment]
Closing this PR and opened a rebased version https://github.com/department-of-veterans-affairs/vets-website/pull/30841

**Pull request closed without merging**